Makefile

```diff
@@ -762,3 +762,32 @@ include tests/integration/tests.mk
 .PHONY: benchcheck
 benchcheck:
 	bin/perfcheck.sh
+
+#-----------------------------------------------------------------------------
+# Target: istio-ecosystem installer
+#-----------------------------------------------------------------------------
+
+export ISTIO_INSTALLER?=${GOPATH}/src/github.com/istio-ecosystem/istio-installer
+INSTALLER_OPTS="-f install/kubernetes/helm/istio/test-values/values-installer-e2e.yaml"
+INSTALLER_REPOSRC=git://github.com/istio-ecosystem/istio-installer.git
+
+force: ;
+
+${GOPATH}/src/github.com/istio-ecosystem/istio-installer: force
+	mkdir -p ${GOPATH}/src/istio-ecosystem
+	git clone "${INSTALLER_REPOSRC}" "${ISTIO_INSTALLER}" 2> /dev/null || (cd "${ISTIO_INSTALLER}" ; git pull)
```

Comment:
should this be the same branch as this repo?

Reply:
perhaps?  it isn't clear to me how _exactly_ we want to manage this relationship.

Reply:
can you add `-b master` so at least it's explicit?

Reply:
done.